app/liquid_tags/github_tag/github_issue_tag.rb

```diff
@@ -46,9 +53,19 @@ def parse_link(link)
     end
 
     def generate_api_link(input)
-      input = input.gsub(/\?.*/, "")
-      input = input.gsub(/\d{1,}#issuecomment-/, "comments/") if input.include?("#issuecomment-")
-      "https://api.github.com/repos/#{input.gsub(/.*github\.com\//, '')}"
+      path = input.gsub(/\?.*/, "")
+      path = path.gsub(/.*github\.com\//, "")
```

Comment:
You can use the `%r{}` regex literals here so you don't have to escape the path slashes:

```suggestion
  path = path.gsub(%r{.*github\.com/}, "")
```

We can also leave off the `if`, since `gsub` doesn't do anything if the first argument doesn't match:

```ruby
"hello rhymes".gsub(%r{/pull/}, "pulls")
#=> "hello rhymes"
```

Reply:
Sure! I only thought it was faster as it doesn't have to scan the entire string, not that it matters being so short :D

Reply:
It would scan the string anyway for the `if`, wouldn't it?